crates/iceberg/src/spec/view_version.rs

```diff
@@ -0,0 +1,313 @@
+    #[inline]
+    pub fn timestamp(&self) -> Result<DateTime<Utc>> {
+        timestamp_ms_to_utc(self.timestamp_ms)
+    }
+
+    /// Get the timestamp of when the view version was created in milliseconds since epoch
+    #[inline]
+    pub fn timestamp_ms(&self) -> i64 {
+        self.timestamp_ms
+    }
+
+    /// Get summary of the view version
+    #[inline]
+    pub fn summary(&self) -> &HashMap<String, String> {
+        &self.summary
+    }
+
+    /// Get this views representations
+    #[inline]
+    pub fn representations(&self) -> &ViewRepresentations {
+        &self.representations
+    }
+
+    /// Get the default catalog for this view version
+    #[inline]
+    pub fn default_catalog(&self) -> Option<&String> {
+        self.default_catalog.as_ref()
+    }
+
+    /// Get the default namespace to use when a reference in the SELECT is a single identifier
+    #[inline]
+    pub fn default_namespace(&self) -> &NamespaceIdent {
+        &self.default_namespace
+    }
+
+    /// Get the schema of this snapshot.
+    pub fn schema(&self, view_metadata: &ViewMetadata) -> Result<SchemaRef> {
+        let r = view_metadata
+            .schema_by_id(self.schema_id())
+            .ok_or_else(|| {
+                Error::new(
+                    ErrorKind::DataInvalid,
+                    format!("Schema with id {} not found", self.schema_id()),
+                )
+            })
+            .cloned();
+        r
+    }
+
+    /// Retrieve the history log entry for this view version.
+    #[allow(dead_code)]
+    pub(crate) fn log(&self) -> ViewVersionLog {
+        ViewVersionLog::new(self.version_id, self.timestamp_ms)
+    }
+}
+
+/// A list of view representations.
+#[derive(Debug, PartialEq, Eq, Clone, Serialize, Deserialize)]
+pub struct ViewRepresentations(pub(crate) Vec<ViewRepresentation>);
+
+impl ViewRepresentations {
+    #[inline]
+    /// Get the number of representations
+    pub fn len(&self) -> usize {
+        self.0.len()
+    }
+
+    #[inline]
+    /// Check if there are no representations
+    pub fn is_empty(&self) -> bool {
+        self.0.is_empty()
+    }
+
+    /// Get an iterator over the representations
+    pub fn iter(&self) -> impl Iterator<Item = &'_ ViewRepresentation> {
+        self.0.iter()
+    }
+}
+
+// Iterator for ViewRepresentations
+impl IntoIterator for ViewRepresentations {
+    type Item = ViewRepresentation;
+    type IntoIter = std::vec::IntoIter<Self::Item>;
+
+    fn into_iter(self) -> Self::IntoIter {
+        self.0.into_iter()
+    }
+}
+
+#[derive(Debug, Serialize, Deserialize, PartialEq, Eq, Clone)]
+#[serde(tag = "type")]
+/// View definitions can be represented in multiple ways.
+/// Representations are documented ways to express a view definition.
+// ToDo: Make unique per Dialect
+pub enum ViewRepresentation {
+    #[serde(rename = "sql")]
+    /// The SQL representation stores the view definition as a SQL SELECT,
+    SqlViewRepresentation(SqlViewRepresentation),
```

Comment:
How about use `Sql` as variant name? So users will use it as `ViewRepresentation::Sql` instead of `ViewRepresentation::SqlViewRepresentation`.

Reply:
Definitely better for usability:
https://github.com/c-thiel/iceberg-rust/commit/bb9cf9c9b290de972fef6953066759a860790cfb